Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -738,7 +737,7 @@ def com_google_fonts_check_029(ttFont, license):
                              "").format(license,
                                         NameID.LICENSE_DESCRIPTION,
                                         nameRecord.platformID,
-                                        PLATID_STR[nameRecord.platformID],
+                                        PlatformID(nameRecord.platformID).name,
```

Comment:
This and other occurances: why not `nameRecord.platformID.name` instead?

Reply:
Because `nameRecord.platformID` is an integer. And ìn order to access the `name` property, we need to instantiate an enum passing the integer value as a constructor.

It is a bit ugly, but I could not find other ways of doing it.

Reply:
Ah, right, it comes from ttFont.

@anthrotype take note, once the Python 3.6 reckoning comes :grimacing: